optimum/bettertransformer/models/base.py

```diff
@@ -91,6 +93,14 @@ def validate_bettertransformer(self):
                 f" Number of heads must be even."
             )
 
+        # Check if norm_first is activated for pytorch==1.12
+        if self.norm_first and not is_pytorch_greater_version("1.13.0"):
```

Comment:
```suggestion
        if self.norm_first == True and not is_pytorch_greater_version("1.13.0"):
```

just personal taste, feel free to ignore